docs/content/guides/author-apps/kubernetes/howto-yaml-base-support/snippets/basemanifest.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-microservice
+  labels:
+    app: my-microservice
+  annotations:
+    source: base-manifest-test
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: my-microservice
+  template:
+    metadata:
+      labels:
+        app: my-microservice
+    spec:
```

Comment:
Would we expect a user to remove the container image from the Deployment and only define it in the Radius container? If so we need to explicitly say that.

Or would we want them to define it in both places? Or can the container image just be defined here, vs. being required to be defined in the container?

Reply:
We can define image in `my-microservices` container, but renderer will override the image location defined in Container resource. 

But if you define image name in the different container like sidecar, container renderer will not override the image location.